android/build.gradle

```
@@ -1,5 +1,5 @@
 buildscript {
-  ext.kotlin_version = '1.2.71'
+  ext.kotlin_version = '1.3.11'
```

```suggestion
  ext.kotlin_version = rootProject.hasProperty('kotlin_version') ? rootProject.kotlin_version : '1.3.11'
```

How about allowing users to specify kotlin version the way we already do with SDK and build tools versions?
So that build.gradle could have something like this
```
buildscript {
    ext {
        buildToolsVersion = "28.0.3"
        minSdkVersion = 21
        compileSdkVersion = 28
        ...
        kotlin_version = "1.3.11"
    }
     ...
}
```